reactos/drivers/usb/usbhub/pdo.c

```diff
@@ -146,6 +146,11 @@ IsValidPDO(
 
     ChildDeviceExtension = (PHUB_CHILDDEVICE_EXTENSION)DeviceObject->DeviceExtension;
     ASSERT(ChildDeviceExtension->Common.IsFDO == FALSE);
+
+    // This can happen when parent device was surprise removed.
```

Comment:
Yes, that will we added in FDO commit set which will be continuation of this one.